site/kubernetes/operator/kubectl-plugin.md

```diff
@@ -0,0 +1,211 @@
+</pre>
+
+### <a id='secrets' class='anchor' href='#secrets'>Print default user secrets</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq secrets INSTANCE
+</pre>
+
+Displays the default user secrets for the named RabbitMQ cluster
+
+### <a id='manage' class='anchor' href='#manage'>Open the Management UI</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq manage INSTANCE
+</pre>
+
+Opens the RabbitMQ Management dashboard in a browser.
+
+### <a id='debug' class='anchor' href='#debug'>Set Log Level to debug</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq debug INSTANCE
+</pre>
+
+Sets the log level on all nodes to debug. For a detailed breakdown on RabbitMQ logging
+see the [logging docs page](/logging.html).
+
+### <a id='tail' class='anchor' href='#tail'>Tail Logs</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq tail INSTANCE
+</pre>
+
+Attach to live log output from all RabbitMQ nodes.
+
+### <a id='observe' class='anchor' href='#observe'>Observe</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq observe INSTANCE INDEX
+</pre>
+
+Opens the rabbitmq-diagnostics observer interface for a given node. For more
+information on the rabbitmq-diagnostics see [here](rabbitmq-diagnostics.8.html).
+
+### <a id='feature-flags' class='anchor' href='#feature-flags'>Enable All Feature Flags</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq enable-all-feature-flags INSTANCE 
+</pre>
+
+This command wraps the `rabbitmqctl` CLI to enable all possible feature flags.
+For comprehensive documentation on all feature flags see [here](/feature-flags.html).
+
+### <a id='pause-reconciliation' class='anchor' href='#pause-reconciliation'>Pause Reconciliation</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq pause-reconciliation INSTANCE 
+</pre>
+
+This adds the label "rabbitmq.com/pauseReconciliation=true" to your RabbitMQ cluster, which prevents the operator from
+watching and updating the instance.
+
+### <a id='resume-reconciliation' class='anchor' href='#resume-reconciliation'>Resume Reconciliation</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq resume-reconciliation INSTANCE 
+</pre>
+
+Instructs the operator to resume reconciliation for a RabbitMQ cluster.
+
+### <a id='list-pause-reconciliation-instances' class='anchor' href='#list-pause-reconciliation-instances'>List Instances with Paused Reconciliation</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq resume-reconciliation INSTANCE 
+</pre>
+
+Lists all instances that are not being reconciled by the operator.
+
+
+### <a id='perf-test' class='anchor' href='#perf-test'>Perf Test</a>
+
+<pre class="lang-bash">
+kubectl rabbitmq perf-test INSTANCE --rate 100
+</pre>
+
+Runs perf-test against an instance. You can pass as many perf test parameters as you like here.
+For more information on perf-test and what parameters are available see [here](https://rabbitmq.github.io/rabbitmq-perf-test/stable/htmlsingle/).
+
+If you want to monitor perf-test, you can do so by creating the following ServiceMonitor:
```

Comment:
We tend to avoid the "you can…", "tutorial style" wording in doc guides. So most guides would say
something like this:

"To monitor PerfTest, add the following ServiceMonitor…"

Reply:
Ok, I will change that now. Thanks Michael.